src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -384,6 +385,25 @@ export default {
 				);
 			}
 		},
+		initializeRecommendedLoansRowExp() {
+			// experiment: CASH-1801 "Loans For You" Recommendation Row
+			// get assignment
+			const recommendationChannel = 0;
+			const recommendedLoansRowEXP = this.apollo.readFragment({
+				id: 'Experiment:recommendation_channel',
+				fragment: experimentVersionFragment,
+			}) || {};
+			this.recommendedLoansRowExpVersion = recommendedLoansRowEXP.version;
+			// Logged in user and non-users are included in this experiment,
+			// logged-in users are automatically tracked with their id
+			// Fire Event for Exp CASH-1801
+			this.$kvTrackEvent(
+				'Lending',
+				'EXP-CASH-1801-Feb2020',
+				recommendationChannel,
+				this.recommendedLoansRowExpVersion === 'shown' ? 'b' : 'a'
```

Comment:
@go-bears recommendationChannel should be the 4th param and version (a or b) should be the 3rd.

Reply:
thanks! will relocate it